react/components/SearchBar/components/ResultsList.js

```diff
@@ -97,15 +103,16 @@ const ResultsList = ({
           <Fragment key={item.name + index}>
             <hr className="o-05 ma0 w-90 center" />
             <Link
-              onClick={() => {
-                onClearInput()
-                closeMenu()
-              }}
+              {...getItemProps({
+                item,
+                onClick: handleItemClick,
+              })}
               {...getLinkProps(item)}
               className={listItemClassNames}
             >
               {item.thumb && (
                 <img
+                  alt=""
```

Comment:
Isn't there an value to set the `alt` avoiding set it with an empty string?

Reply:
The linter encourages setting it to an empty string when the image is decorative (this case has the product name right after the image, so I don't think there was much to gain from adding any alt text to the image)